corehq/apps/app_manager/templates/app_manager/partials/modules/case_list_form_setting.html

```diff
@@ -74,6 +74,20 @@
         </div>
       {% endif %}
     </div>
+    {% if request|toggle_enabled:'FOLLOWUP_FORMS_AS_CASE_LIST_FORM' %}
+      <div class="form-group" id="case_list_form-expression" data-bind="visible: caseListForm()">
+        <label class="{% css_label_class %} control-label">
+          {% trans "XPath Expression to Enable Case List Form" %}
+          <span class="hq-help-template"
+                data-title="{% trans "XPath Expression" %}"
```

Comment:
This and the next line should use `trans_html_attr` so they handle quotes correctly.